Makefile

```diff
@@ -36,13 +36,20 @@ format:
 install-cert:
 	scripts\install_cert.bat ${cert} ${pass}
 
-## install-scanner - Install SecurityCodeScan to your system
-install-scanner:
-	dotnet tool install --local security-scan --version 5.6.3
+## install-tools - Install required dotnet tools
+install-tools:
+	dotnet new tool-manifest || true
+	dotnet tool install --local security-scan --version 5.6.3 || true
+	dotnet tool install --local dotnet-format || true
```

Comment:
Prefer `exit 0` for fallbacks as it more clearly defines our expected outcome.

Reply:
Resolved